account_statement_import_camt/models/res_config_settings.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 CampToCamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+MODULE = "account_statement_import_camt"
+CHECKING_NAME_UNIQUE = MODULE + ".checking_bank_statement_name_unique"
+
+
+class ResConfigSetting(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    checking_bank_statement_name_unique = fields.Boolean(
+        string="Unique bank statement name",
```

Comment:
Because the scope of the parameter is limited to CAMT statements, I would prefer it if tthe field name and string referred to CAMT to make this clear to the user.

Reply:
@hailangvn can you address this please? So we can merge :)

Reply:
Thanks @StefanRijnhart for your suggestion. I agree that the field name and string should reflect the logic. But in the code to check duplicated names, I could not find any logic specific to camt. The code only check for generic field 'name'. It may confuses because it is in camt module and can be split to another module. But that work is for future when we see another module needs the logic.

Reply:
OK, we clearly don't agree. I'll abstain from further review.